networkx/algorithms/traversal/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .beamsearch import *
 from .breadth_first_search import *
 from .depth_first_search import *
+from .depth_limited_search import *
```

Comment:
I added it by mistake when I was squashing the many previous commits while rebasing. Thanks for catching that.